Open Judge System/Workers/OJS.Workers.LocalWorker/OJS.Workers.LocalWorker.csproj

```diff
@@ -58,6 +58,10 @@
     <Reference Include="log4net, Version=2.0.8.0, Culture=neutral, PublicKeyToken=669e0ddf0bb1aa2a, processorArchitecture=MSIL">
       <HintPath>..\..\packages\log4net.2.0.8\lib\net45-full\log4net.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.AspNet.Identity.EntityFramework, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35" />
```

Comment:
Is this reference necessary?

Reply:
Yes, without this I cannot Bind IOjsDbContext to OjsDbContext in the Main method of the worker